cmd/login/cmd.go

```diff
@@ -164,6 +169,21 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	// Load the configuration file:
+	cfg, err := config.Load()
+	if err != nil {
+		r.Reporter.Errorf("Failed to load config file: %v", err)
+		os.Exit(1)
+	}
+	if cfg == nil {
```

Comment:
@tirthct You don't need this check as the `config.Load()` method will always return an empty `Config` if one doesn't exist.

Reply:
I don't think that check would be working properly. I removed it and it stopped me from logging in:
![image](https://github.com/openshift/rosa/assets/85271178/33fa0cc0-5bfa-4b14-bd89-c62b21429c83)

Not looking into that in this PR this change is already in the master branch for long, but def something to look into.

Reply:
PS. ignore the double warnings. For some reason my machine sometimes does that. 
@tylercreller has verified and confirmed the message is only shown once